internal/blocktx/server.go

```diff
@@ -3,13 +3,13 @@ package blocktx
 import (
 	"context"
 	"errors"
+	grpcopts "github.com/bitcoin-sv/arc/internal/grpc_opts"
```

Comment:
No need for alias as rule has been excluded

Reply:
Removed